packages/wonder-blocks-tokens/src/tokens/color.ts

```diff
@@ -98,4 +98,40 @@ export const color = {
     fadedOffBlack32: fadedColorWithWhite(offBlack, 0.32),
     fadedOffBlack16: fadedColorWithWhite(offBlack, 0.16),
     fadedOffBlack8: fadedColorWithWhite(offBlack, 0.08),
+
+    // Magenta - same as Thunderblocks
```

Comment:
We want to use the same hex values for gem/streak/due semantic core color tokens in the classic theme. Let me know if there's a better approach for this! I've copied over the new colors to classic. For more context, see [thread about colors](https://khanacademy.slack.com/archives/C07CA2YR0BZ/p1747668828262109)!

Other options
- import thunderblocks primitives and use it here
- hardcode the hex values in `semanticColor.ts`
- something else?

Reply:
I think it would be fine importing from `theme/semantic/internal/primitive-color-thunderblocks.ts` here. That would help centralizing those color values in case we want to change them in the future.